pkg/epp/server/runserver.go

```diff
@@ -45,13 +46,16 @@ type ExtProcServerRunner struct {
 	DestinationEndpointHintKey               string
 	PoolName                                 string
 	PoolNamespace                            string
-	RefreshMetricsInterval                   time.Duration
-	RefreshPrometheusMetricsInterval         time.Duration
 	Datastore                                datastore.Datastore
-	Provider                                 *backend.Provider
 	SecureServing                            bool
 	CertPath                                 string
 	UseStreaming                             bool
+	RefreshPrometheusMetricsInterval         time.Duration
+
+	// This should only be used in tests. We won't need this once we don't inject pods into the
```

Comment:
We are not injecting pods into the store in tests now, we changed that already, this injects metrics only.